### DISCUSSION TITLE
Alert: "outline" style variants

### DISCUSSION THREAD

**Discussion by dhedegaard-digisense** [saadeghi/daisyui#2197] [Ideas / feature requests]
The `Alert` component has various styling options useful to indicate success, error, warning and info to the user.

https://dev.daisyui.com/components/alert/

Would it make sense to add some outline variants similar to `Button` that uses a light background and uses the chosen color primarily in the border?

**Comment by saadeghi** [comment]
It's a good idea.  
I will work on it as soon as I'm done with fixing other issues.  
Currently fixing the issues is more important than adding new features.

Thank you 🙏

**Comment by stefanobartoletti** [comment]
I second this request. It would be very handy to have a preset for outlined alerts.

Is this still planned?
